LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012 Miki Tebeka <miki.tebeka@gmail.com>.
+Copyright (c) 2012 Miki Tebeka <miki.tebeka@gmail.com>, Google Inc.
```

Comment:
Google owns the copyright on this patch, so any patches employees make need to include that line. An entry in an AUTHORS file is fine too.